packages/manager/modules/nutanix/src/dashboard/general-info/redeploy/template.html

```diff
@@ -328,13 +339,21 @@ <h2 data-translate="nutanix_redeploy_title"></h2>
                                     data-ng-model-options="{ debounce: 500 }"
                                     required
                                     data-ng-pattern="$ctrl.IPV4_REGEX"
+                                    data-unique-ip-validator="$ctrl.getIpsList()"
                                     data-ip-subnet-validator="$ctrl.config.gatewayCidr"
                                 />
                                 <small
                                     class="form-text text-danger"
                                     data-ng-if="$ctrl.displayIpSubnetErrorPrismCentralIpField(customConfig, $index)"
                                     data-translate="nutanix_redeploy_custom_config_ip_not_in_cidr_subnet_error"
-                                ></small>
+                                >
+                                </small>
+                                <small
+                                    class="form-text text-danger"
+                                    data-ng-if="$ctrl.displayIpUniqueErrorPrismCentralIpField(customConfig, $index)"
```

Comment:
better to prefix function name with `is`/`can` to avoid confusion.

Reply:
Done https://github.com/ovh/manager/commit/798cf2f71c0f75d40fd7d8f26284603a7293d07a